src/test/java/com/williamfiset/algorithms/sorting/RadixSortTest.java

```diff
@@ -0,0 +1,20 @@
+package com.williamfiset.algorithms.sorting;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.Test;
+
+public class RadixSortTest {
+  @Test
+  public void testGetMax() {
```

Comment:
Can you add more test coverage? Can you also add some randomized tests? E.g:

/src/main/java/com/williamfiset/algorithms/sorting/SelectionSort.java@master#L45

Reply:
Random Test added